.pipelines/build-pipelines.yml

```diff
@@ -211,7 +211,7 @@ steps:
   inputs:
     command: 'push'
     feedsToUse: 'select'
-    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg'
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
```

Comment:
i thought `.symbols.nupkg` files invalid and are not generated anymore

Reply:
remnant of git revert. removed it